Model/Method/Cc.php

```diff
@@ -242,18 +241,19 @@ public function assignData(\Magento\Framework\DataObject $data)
                 date(
                     'd/m/Y',
                     strtotime(
-                        $this->pagSeguroHelper->getCCOwnerData('credit_card_birthyear').
+                        $data['additional_data']['cc_owner_birthday_day'].
```

Comment:
This would return false if customer type a day > 12 and the wrong date would be sent.
I fixed it already here. Thanks.
See https://www.php.net/manual/en/datetime.formats.php for details.